src/04 - Web services.md

```diff
@@ -103,58 +116,230 @@ Content-Type: application/json
 ###### Details
 
 - **results**
-  - array of one or more code lists
-- **type**
-  - object type that this code list applies to
-- **field**
-  - field in the object type that this code list applies to
-- **code**
-  - code or short name of the code list value
-- **name**
-  - user-friendly name of the code list value
-- **description**
-  - description of the code list value
+  - array of zero or more groups of business-specific metadata field definitions
+  - **groupId**
+    - system unique group ID (a string comprised of lower-case letters, digits, and dashes that must begin with a letter and end with a letter or a digit)
+  - **groupName**
+    - group name
+  - **groupDescription**
+    - group description
+  - **fields**
+    - array of zero or more business-specific metadata field definitions in the particular group
+    - **fieldId**
+      - system unique field ID (a string comprised of lower-case letters, digits, and dashes that must begin with a letter and end with a letter or a digit)
+    - **fieldName**
+      - field name
+    - **fieldDescription**
+      - field description
+    - **fieldType**
+      - field type (possible values are string, long, and double)
+    - **fieldValues** (optional)
+      - array of field values of the particular field type
+      - will be returned only for controlled fields (an array of zero or more values); will not be returned for simple fields
```

Comment:
I would assume the terms `controlled fields` and `simple fields` would mean little to integrating parties.

I'd suggest re-wording this to:
```
- if present, an array of zero or more values
- will be returned only for fields with pre-defined values (i.e. values assigned to the field upon its creation)
```

The same occurs in at least one more location in this document too.

Reply:
Fixed.